specification/eventgrid/data-plane/Microsoft.ContainerService/stable/2018-01-01/ContainerService.json

```diff
@@ -27,6 +27,67 @@
           "description": "The highest PATCH Kubernetes version considered preview for the ManagedCluster resource. There might not be any version in preview at the time of publishing the event",
           "type": "string"
         }
+      },
+      "x-ms-examples": {
+        "newKubernetesVersionAvailableCloudEventsSchema": {
+          "$ref": "./examples/cloud-events-schema/new_kubernetes_version_available.json"
+        },
+        "newKubernetesVersionAvailableEventGridSchema": {
+          "$ref": "./examples/event-grid-schema/new_kubernetes_version_available.json"
+        }
+      }
+    },
+    "ContainerServiceClusterOutOfSupportEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.ContainerService.ClusterOutOfSupport event",
+      "type": "object",
+      "properties": {
+        "kubernetesVersion": {
+          "description": "The Kubernetes version of the ManagedCluster resource",
+          "type": "string"
+        },
+        "isOutOfSupport": {
+          "description": "Indicates if the Kubernetes version of the ManagedCluster resource is out of support",
+          "type": "boolean"
+        }
+      },
+      "x-ms-examples": {
+        "clusterOutOfSupportCloudEventsSchema": {
+          "$ref": "./examples/cloud-events-schema/cluster_out_of_support.json"
+        },
+        "clusterOutOfSupportEventGridSchema": {
+          "$ref": "./examples/event-grid-schema/cluster_out_of_support.json"
+        }
+      }
+    },
+    "ContainerServiceNodePoolRollingProgressEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.ContainerService.NodePoolRollingProgress event",
+      "type": "object",
+      "properties": {
+        "nodePoolName": {
+          "description": "The name of the node pool in the ManagedCluster resource",
+          "type": "string"
+        },
+        "progressState": {
+          "description": "Indicates the progress of the node pool rolling process. Possible values are: \"Started\", \"Succeeded\", \"Failed\"",
+          "type": "string",
+          "enum": [
+            "Started",
+            "Succeeded",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "progressState",
```

Comment:
This name is probably too generic - what about something like nodePoolProgressState?

Reply:
Updated it to rollingProgressState, let me know if you feel it's still too generic